### DISCUSSION TITLE
Curious regarding validation loss plot not being in segmentation tutrorials

### DISCUSSION THREAD

**Discussion by muk465** [Project-MONAI/MONAI#4629] [Q&A]
I recently saw tutorials of unetr segmentation ,swin unetr segmentation and spleen segmentation tutorial but in none of them i could see validation loss plot ,isn't validation loss important to check for overfitting? but i couldnt find it in any tutorial and neither in swinunetr paper

**Comment by JohnMasoner** [comment]
Hi @muk465 ,

The validation loss no backward, it's only used as an indicator similar to the validation  accuracy to change teh learning rate. So many people used the dice coefficient as the indicator, it's better in segmentation task.

Sure, you can use the validation losss as the indicator to change the learning rate when you only have low-shot datasets.

Thanks
